veevavault/1.3/modules/ROOT/pages/veevavault-connector-config-topics.adoc

```diff
@@ -1,622 +1,38 @@
+The *Invoke Rest API* operation sends the Mule message payload as the request body.
 
-[cols=",",options="header",]
-|===
-|*Field* |*Description*
-|*Method*|Method to invoke the Veeva Vault Rest API. The default is *GET*.
-|*Path*|URI path to invoke the Veeva Vault Rest API. For example: `/objects/documents/{doc_id}`
-|*Body*|Body to send with a request to invoke Veeva Rest API. Default is *#[payload]*.
-|*Headers*|Headers in key-value format to send along with the request. If there are no required headers in the request, leave empty. The default is *Empty list*.
-|*URI Parameters*|URI Parameters in key-value format to resolve the value in the path. If there are no URI parameters in the path, leave empty. The default is an *Empty List*.
-|*Query Parameters*|Query parameters in key-value format to send along with the request. If there are no query parameters required in the request, leave empty. The default is *Empty list*.
-|*Request Streaming Mode*|Whether to use streaming mode for the request. By default, if the type of the payload is a stream, streaming is used to send the request. Select the streaming option from the drop-down list.
-|*Input*|N/A
-|*Output*|Response either in JSON (default) or XML format, depending on the specified header *Accept* value provided in *Headers* section.
-|===
+In addition to the request body, you can also configure the following using a DataWeave script or expression.
 
-A typical use of the Invoke Rest API operation is to invoke the Veeva Vault Rest API using the default GET method. By default, GET methods do not send the payload in the request; the body of the HTTP request is empty. The other methods send the message payload as the body of your request.
+* <<headers>>
+* <<uri-parameters>>
+* <<query-parameters>>
 
-After sending a request, the connector receives the response either in JSON (default) or XML format, depending on the header *Accept* value provided. The payload is then passed to the next element in your application’s flow.
 
-==== Add Parameters
+[[headers]]
+=== Headers
 
-By default, the Invoke Rest API operation sends the Mule message payload as the request body, but you can customize it using a DataWeave script or expression. In addition to the body of the request, you can configure:
+In *General > Request > Query Parameters*, click the plus icon (+) to add headers to the request.
 
-- Headers
-- Query parameters
-- URI parameters
-
-==== Headers
-
-In *General > Request > Query Parameters*, click the plus icon (+) to add headers to the request. For example, add header names `HeaderName1` and `HeaderName2`, and header values `HeaderValue1` and `HeaderValue2`.
-
-You can use DataWeave expressions, for example:
+For example, you can add header names `HeaderName1` and `HeaderName2`, and header values `HeaderValue1` and `HeaderValue2`, by using DataWeave expressions:
 
 ----
 #[{'HeaderName1' : 'HeaderValue1', 'HeaderName2' : 'HeaderValue2'}].
 ----
 
```

Comment:
If you decide to make numbered steps of this topic, rework to fit in this example.